### ISSUE TITLE
Deprecate `@testing-library/preact-hooks`

### ISSUE THREAD

**Issue by MichaelDeBoey** [testing-library/preact-hooks-testing-library#17]
Now that https://github.com/testing-library/preact-testing-library/issues/56 is merged, this library can be deprecated

CC/ @testing-library/preact-hooks

**Comment by JoviDeCroock** [comment]
I don't think so, even for react they are kept around, let's not deviate too hard 😅 `@testing-library/preact-hooks` does a lot more as well in  terms of Suspense and so on

**Comment by nickserv** [comment]
Any more opinions on this? I think we should at least add a blurb to the readme like we did for React Testing Library.

**Comment by kentcdodds** [comment]
An explanation in the README makes sense to me. If this library does more than the other, maybe those features should be added to the other so this can be deprecated.

I don't have any basis for opinion here. Just some thoughts.
